code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -515,6 +515,7 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	var/used_message = "<span class='holoparasite'>All the cards seem to be blank now.</span>"
 	var/failure_message = "<span class='holoparasite bold'>..And draw a card! It's...blank? Maybe you should try again later.</span>"
 	var/ling_failure = "<span class='holoparasite bold'>The deck refuses to respond to a souless creature such as you.</span>"
+	var/mori_message = "<span class='holoparasite bold'>The deck shows the Fool! It quickly disappears though, as you need lifeforce to sustain it...</span>"
```

Comment:
document this

Reply:
I am sorry, what do you mean by document this, where would I document it?

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md

Reply:
That sadly didn't help me understand at all, maybe I am a denthead, or it is also possible that I should be usint linux to code this.

Reply:
Use autodocumentation to document the new variable, as described by that guide